collections/Fragments_printed_books/Arch_A_f_131.xml

```diff
@@ -0,0 +1,110 @@
+                              <height>145</height>
+                              <width>94</width>
+                           </dimensions>
+                        </extent>
+                     </supportDesc>
+                     <layoutDesc>
+                        <layout ruledLines="22">22 long lines.</layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote scope="major" script="minusculeInsular">Insular minuscule.</handNote>
+                     <handNote scope="minor" script="minusculeCaroline">Gloss in Anglo-Caroline.</handNote>
+                  </handDesc>
+                  <decoDesc>
+                     <decoNote type="decInit">Minor decorated initial.</decoNote>
+                  </decoDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notAfter="0810" notBefore="0790">8th century, late, or 9th century, early</origDate>; <origDate type="additions" notAfter="1050" notBefore="1000" calendar="Gregorian">glosses, 11th century, first half (?)</origDate>
+                     <origPlace>
+                        <country key="place_7002445">English</country>, perhaps <settlement key="place_7015155">Worcester</settlement> or <settlement key="place_7012044">Canterbury</settlement> (?)</origPlace>
+                  </origin>
+                  <provenance>For the date and origin see the conspectus of earlier scholarship in <bibl>Dieter Studer-Joho, <title>A catalogue of manuscripts known to contain Old English dry-point glosses</title> (2017), no. 24</bibl>.</provenance>
+                  <provenance>In the area of Kent by the second half of the tenth century, as shown by added Old English glosses in Kentish dialect on other leaves of the manuscript (<bibl>Ker, <title>Catalogue</title>, no. 12</bibl>).</provenance>
+                  <provenance>Apparently broken up in Brighton in the early nineteenth century: see Gwara, op. cit., pp. 86-7, with details of the history of other leaves.</provenance>
+                  <acquisition when="1965">The present leaf acquired as the wrapper of W. Perkins, <title>Satans sophistrie answered</title> (1604) from Blackwells in 1965 (<bibl>N. Ker, ‘Supplement to Catalogue of Manuscripts Containing Anglo-Saxon’, <title>Anglo-Saxon England</title> 5 (1976), 122</bibl>).</acquisition>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>Description by Matthew Holford, March 2020.</source>
+                     </recordHist>
+                  </adminInfo>
+               </additional>
+            </msDesc>
+         </sourceDesc>
+      </fileDesc>
+      <revisionDesc>          <change when="2020-03-09">First online publication.</change>
+
+         <change when="2017-05-29">
```

Comment:
I assume these 2017 lines shouldn't be there?

Reply:
good point!